internal/clients/client.go

```diff
@@ -331,7 +331,9 @@ func (client *Client) Build(ctx context.Context, o *common.ClientOptions) error
 	if client.ConfidentialLedger, err = confidentialledger.NewClient(o); err != nil {
 		return fmt.Errorf("building clients for ConfidentialLedger: %+v", err)
 	}
-	client.Connections = connections.NewClient(o)
+	if client.Connections, err = connections.NewClient(o); err != nil {
+		return fmt.Errorf("building clinets for Connections: %+v", err)
```

Comment:
This is misspelled everywhere, please fix these 
```suggestion
		return fmt.Errorf("building clients for Connections: %+v", err)
```

Reply:
sorry it's a typo when recording the macro. updated now.